client/src/components/NavbarComponent.js

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+export function Navbar() {
+  return (
+    <nav className="navbar  navbar-expand-lg  shadow-lg">
+      <div className="navbar-collapse" id="navbarsExampleDefault">
+        <nav className="nav nav-underline">
+          <span className="nav-link">
+            <Link to="/home"><img src={'https://image.flaticon.com/icons/svg/25/25231.svg'} alt={'go to home'} height='24' /></Link>
+          </span>
+          <span className="nav-link">
```

Comment:
I put About for just as a reference. Actually there is no need to display in the `Navbar`.

Reply:
What' the utility of an empty Page? not linked in the Navbar? useless code

Reply:
Yeap, it is a useless code. I used this project to learn to React. Actually, this was my first React app. I used those to study how to React router works with several pages.